src/load/azext_load/data_plane/load_trigger/custom.py

```diff
@@ -0,0 +1,233 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_load.data_plane.utils.utils import (
+    get_admin_data_plane_client,
+    )
+
+from azure.core.exceptions import ResourceNotFoundError
+from knack.log import get_logger
+from azext_load.vendored_sdks.loadtesting.models import ( _models as models, _enums as enums)
+import azext_load.data_plane.load_trigger.utils as utils
+
+logger = get_logger(__name__)
+
+def create_trigger_schedule(
+    cmd,
+    load_test_resource,
+    resource_group_name=None,
+    trigger_id=None,
+    description=None,
+    display_name=None,
+    trigger_start_date_time=None,
+    recurrence_type=None,
+    recurrence_interval=None,
+    recurrence_index=None,
+    recurrence_cron_expression=None,
+    recurrence_dates_in_month=None,
+    recurrence_week_days=None,
+    end_after_occurrence=None,
+    end_after_date_time=None,
+    test_ids=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.info("Creating trigger schedule")
```

Comment:
Should we validate trigger doesnt exist ?

Reply:
Should we say schedule trigger everywhere?